velox/exec/HashTable.cpp

```diff
@@ -1201,7 +1204,8 @@ void HashTable<ignoreNullKeys>::rehash() {
       numGroups = (i == 0 ? this : otherTables_[i - 1].get())
                       ->rows()
                       ->listRows(&iterator, kHashBatchSize, groups);
-      if (!insertBatch(groups, numGroups, hashes)) {
+      if (!insertBatch(
+              groups, numGroups, hashes, initNormalizedKeys || i != 0)) {
```

Comment:
Why need to re-compute normalized keys if insert to other tables if initNormalizedKeys is false? thanks!

Reply:
@zhli1142015 Zhen, would you take a look at this question from @xiaoxmeng ?

Reply:
Hello @xiaoxmeng and @mbasmanova ,
I thought rows from `otherTables_` should be re-compute. Looks there is no need to re-compute them also. But from my understanding `otherTables_` is only used for parallel join building, so there is no impact to this PR (aggregation). Please let me know if this is correct and I should remove `1 != 0` from line 1208 by another PR.
Thanks.

Reply:
@zhli1142015 thanks for the explanation. It seems in this case, it really depends on the actual use case to compute hash for other tables or not. How about we add a comment there about this without actual code change. Thanks!

Reply:
Thanks, I'm ok with this.